src/main/java/emissary/util/DataUtil.java

```diff
@@ -50,15 +50,15 @@ public static boolean isNotEmpty(@Nullable final IBaseDataObject d) {
     }
 
     /**
-     * Return true if the data slot is empty or just whitespace and/or control chars
+     * Return true if the data slot is empty or just blank space and/or control chars
```

Comment:
This is _(and already was)_ a misleading description of the functionality.  The underlying code doesn't examine individual characters at all:
```
    public static boolean isEmpty(@Nullable final byte[] data) {
        return (data == null) || (data.length == 0);
    }
```